.buildkite/test/run.sh

```diff
@@ -42,11 +42,4 @@ pnpm run test
 
 # New client test suite
 # 
-# TODO make make side effect free and isolated since it can right now, only be ran after `pnpm run test`
-# because it drops the postgresql `tests` database which result in the following error during `pnpm run test` if `test:functional` are run before
-# FAIL  src/__tests__/integration/postgresql/runtime.test.ts
-# FAIL  src/__tests__/integration/postgresql/introspection.test.ts
-#  error: database "tests" does not exist
-# Test Suites: 2 failed, 8 passed, 10 total
-# https://buildkite.com/prisma/release-prisma-typescript/builds/6514
```

Comment:
Then you can now move `pnpm run --filter "@prisma/client" test:functional` to be before `pnpm run test`